{{cookiecutter.project_name}}/Makefile

```diff
@@ -52,12 +52,14 @@ setup: setup-checks ## install dev requirements
 	virtualenv --python=$(PYTHONVERSION) $(PYTHONENV)
 	$(ENVPIP) install -r requirements_dev.txt
 
+clean-all: clean
```

Comment:
I think clean should always be clean all.

Reply:
Can't be because otherwise some commands will fail due to missing virtualenv (eg. dist target)